src/main/msp/msp_build_info.c

```diff
@@ -35,7 +35,7 @@
 
 #include "msp/msp_build_info.h"
 
-void sbufWriteBuildInfoFlags(sbuf_t *dst)
+void sbufWriteBuildInfoFlags(sbuf_t *dst) 
```

Comment:
```suggestion
void sbufWriteBuildInfoFlags(sbuf_t *dst)
```

Reply:
Rebased PR and started over with new run - this is still included :)

Reply:
it is in the `.py` file, i will PR although not important.

Reply:
:space_invader: